main_test.go

```diff
@@ -34,15 +34,14 @@ func init() {
 
 	flag.BoolVar(&runAcceptanceTests, "acceptance-tests", false, "set flag when running acceptance tests")
 	flag.StringVar(&cliArgsRaw, "cli-args", "", "for passing urfave/cli args (single string) when golang flags are specified (avoids conflict)")
-
-	flag.Parse()
 }
 
 func TestRunMain(t *testing.T) {
 	if !runAcceptanceTests {
 		t.Skip()
 	}
 
+	flag.Parse()
```

Comment:
Can you move this statement before the if test above before you merge? The above test will always evaluate to false otherwise.

Reply:
argh, right - sorry that was a pretty mindless shift of the call 😞 - should be fixed now 👌